### DISCUSSION TITLE
Branch Name wechseln

### DISCUSSION THREAD

**Discussion by Mersag** [neuefische/questions#72] [Web] [answered]
Guten Morgen, 
ich möchte meinen Branch Namen für die User Story 1 wechseln und erhalte eine Fehlermeldung.
 - ### Was muss ich tun? Vielen Dank im voraus :-)
Bereits getan: Terminal alter Name = git checkout **week1-day1** neuer Name =  anschlissend git branch -m **Yoga List-db** oder git branch -m **exercises list**![image](https://user-images.githubusercontent.com/107836739/185038743-d7d0251b-c373-44fd-86d8-3b8afc4d66ac.png)
Vielen Dank im voraus :-)

**Comment by Seborider** [comment, marked as answer]
Guten Morgen! Ich denke es liegt daran, dass du deinen Branchnamen in einem Wort schreiben musst. Also z.B. exercises_list, anstatt exercises list.

**Reply by Seborider** [reply]
https://stackoverflow.com/questions/6591213/how-do-i-rename-a-local-git-branch
git denkt sonst, du willst den branch exercises in list umbenennen und gibt dir die Fehlermeldung, dass exercises gar nicht existiert.

**Reply by Mersag** [reply]
Stimmt . Danke :-)

**Comment by tsimvris** [comment]
Versuch mit git branch -m week1-day1 exercise-list

**Reply by Mersag** [reply]
Hat geklappt. Danke :-)
